docs-requirements.txt

```diff
@@ -24,6 +24,7 @@ asyncpg>=0.12.0
 boto~=2.0
 botocore~=1.0
 celery>=4.0
+elasticsearch>=2.0
```

Comment:
Isn't there a max version for this? This kind of pinning led us to build quite sometimes when a new major version is release with breaking changes.

Reply:
I agree but wanted this to match [pyproject.toml](https://github.com/open-telemetry/opentelemetry-python-contrib/blob/6d1c740effe8c9d9a1b65e0d77497d9cefdd805e/instrumentation/opentelemetry-instrumentation-elasticsearch/pyproject.toml#L36). This might be out-of-scope for this PR but I suggest limiting elasticsearch v for both places WDYT?